data-prepper-api/src/main/java/org/opensearch/dataprepper/model/sink/AbstractSink.java

```diff
@@ -17,14 +17,25 @@
  * This class implements the Sink interface and records boilerplate metrics
  */
 public abstract class AbstractSink<T extends Record<?>> implements Sink<T> {
+    static private final String NUM_RETRIES = "num_retries";
```

Comment:
Please rename to `number_retries` to support our configuration consistency efforts. Using full words helps maintain consistency.

Reply:
As mentioned above. I could remove this completely and use a fixed (not configurable) value.